scripts/transact/connected_performTransfers.ts

```diff
@@ -0,0 +1,148 @@
+import {BigNumber, Wallet} from 'ethers';
+import fs from 'fs-extra';
+import hre, {ethers, network} from 'hardhat';
+import {waitFor} from '../utils/utils';
+// import {AddressZero} from '@ethersproject/constants';
+
+async function main() {
+  const readOnly = network.name !== 'hardhat';
+
+  // const {deployer} = await ethers.getNamedSigners();
+  // const ethFaucet = await ethers.getSigner(AddressZero); // FAILS WITH : The nonce generation function failed, or the private key was invalid
```

Comment:
Is this working now?

Reply:
This does not work. on a hardhat fork you can impersonate any account but this would not work with the zero address, at least via ethers